src/server/pkg/storage/chunk/storage.go

```diff
@@ -37,75 +38,28 @@ func NewStorage(objClient obj.Client, opts ...StorageOption) *Storage {
 
 // NewReader creates a new Reader.
 func (s *Storage) NewReader(ctx context.Context, dataRefs []*DataRef) *Reader {
-	return newReader(ctx, s.objClient, dataRefs)
+	// using the empty chunkset for the reader
```

Comment:
This comment seems unnecessary.

Reply:
This is worth noting because the empty string disables the renewer.  Where should we document that so someone doesn't come along and add a more descriptive string trying to match what we do for the writer?

Reply:
I'm just saying the comment does not tell you any more than you can infer from the usage. You could add a comment that says why we use the empty chunk set, which could provide some value.